rootio/tchain.go

```diff
@@ -0,0 +1,115 @@
+package rootio
+
+type tchain struct {
+	trees []Tree
+}
+
+func (tchain) Class() string {
+	return "TChain"
+}
+
+// Chain returns a tchain that is the concatenation of all the input Trees.
+func Chain(trees ...Tree) tchain {
+	var t tchain
+	t.trees = append(t.trees, trees...)
+	return t
+}
+
+func (t tchain) Entries() int64 {
+	var v int64 = 0
```

Comment:
you can drop the ` = 0` part.
in Go, all values which are not initialized get a default zero value.
that zero value depends on the type, of course (_e.g._ for a string, that's `""`, for a pointer it's `nil`, for a `float64`, it's `0.0`, _etc..._)